gym_xiangqi/envs/xiangqi_env.py

```diff
@@ -1,13 +1,170 @@
 import gym
-# from gym import error, spaces, utils
-# from gym.utils import seeding
+from gym import spaces
+import numpy as np
+
+from gym_xiangqi.xiangqi_game import XiangQiGame
+from gym_xiangqi.piece import Piece, General, Advisor, Elephant
+from gym_xiangqi.piece import Horse, Chariot, Cannon, Soldier
+
+
+PIECE_POINTS = [
```

Comment:
Not a big deal, and feel free to do it in another PR if needed. But I would prefer if general constants that might be used in a lot of places be put in another file instead.

Reply:
changed made based on this